sphinx/source/docs/user_guide/graphs.rst

```diff
@@ -0,0 +1,202 @@
+.. _userguide_graphs:
+
+Visualizing Network Graphs
+==========================
+
+Bokeh has added native support for creating network graph visualizations with
+configurable interactions between edges and nodes.
+
+Edge and Node Renderers
+-----------------------
+
+The secret sauce of the new GraphRenderer is that it maintains separate
```

Comment:
For the sake of non-native English speaking users, I'm going to suggest boring ("key feature of...") over idiomatic. Also no need for "new" since it won't be new for long :)